CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ### Bug fixes
 
+- Add retry feature for locating a package URI on install-unprivileged.sh
```

Comment:
Actually since install-unprivileged.sh is always run from the main branch, it doesn't really make sense to have a CHANGELOG entry for it, which gets associated with a release.  So please remove this  new entry you added.  For the same reason there's no need to backport it to the release-1.1 branch.

Reply:
Thank you @DrDaveD! I understand.